README.md

```diff
@@ -2,162 +2,102 @@
 
 Time series box, line plots and heatmaps for every available season
 
-## Installation
 
-### Easy deploy with Docker
+### Set up a local development environment
 
-```sh
-git clone https://github.com/terraref/traitvis-webapp
-cd traitvis-webapp
-
-docker build -t shiny-traits .
-## after deploying app, launch so it can be accessed by browser at localhost:3838
-docker run --rm -t -i -p 3838:3838 shiny-traits
-```
+#### Database connection
 
-To set one of the environment variables (see "Setup and Notes" section), you can use the `-e` flag, for example to connect to the TERRA REF trait database (bety6), run:
+This application requires a connection to a BETYdb database. It requires a BETYdb database that uses the experiments table to associate time ranges and sites or plots with specific experiments or seasons.
 
-```sh
-docker run --rm -t -i -e bety_host=bety6 -p 3838:3838 shiny-traits
-```
+A connection to a local instance of BETYdb can be created using the TERRA-REF BRAPI API. Clone the [terraref/brapi repository](https://github.com/terraref/brapi) and run Docker commands to create a connection. 
 
-To enter the container bash shell as root, e.g. for development, testing, reviewing logs:
+Clone terraref/brapi repository:
```

Comment:
Would it make sense to just put a copy of the docker-compose.override.yml from brapi into this repository? It seems like a lot of work to clone a whole other repository just to get the docker compose file.

Reply:
@dlebauer yes, I think it would be a good idea to put the copy of the docker-compose.override.yml in the brapi repository.

But, I am confused about the second part of your comment. Wouldn't the user have to clone the brapi repository to set up the brapi docker in order to connect to the database? Is there another way to do it?

Reply:
Have you tried to run the commands with the docker-compose file? What if you just removed lines 42:60 of https://github.com/terraref/brapi/blob/master/docker-compose.yml#L42?

Reply:
The commands in the docker-compose file work (without removing any lines). Instead of cloning the brapi repo, I will copy the docker-compose.yml and docker-compose.override.yml into this repo.

Reply:
The lines for running brapi and brava don't make sense in this repo and should be removed to avoid confusion.

Reply:
Okay